wallet/src/actors/worker/handlers/create_wallet.rs

```diff
@@ -14,6 +14,8 @@ pub struct CreateWallet {
     pub seed_source: types::SeedSource,
     /// Overwrite flag
     pub overwrite: bool,
+    /// Wallet creation checkpoint
```

Comment:
Let's try to deprecate the "checkpoint" term and rather use "protocol epoch". In that way, we make things simpler by having only one way to name stuff, and maybe we'll be able to reuse the "checkpoint" term in the future for proper chain checkpointing (expedited synchronization).
```suggestion
    /// Protocol epoch in which a wallet was created (won't synchronize blocks prior to this epoch)
```